snapcraft/internal/pluginhandler/__init__.py

```diff
@@ -333,9 +334,24 @@ def pull(self, force=False):
     def mark_pull_done(self):
         pull_properties = self.code.get_pull_properties()
 
+        # Add the annotated list of build packages
```

Comment:
Ah ha, that's what I'm talking about! However, at this point, is there any advantage to actually storing it in the class instance at all? Or should we just store it in a temporary variable here just to shove into the state?